src/nanaimo/fixtures.py

```diff
@@ -578,6 +578,10 @@ class SubprocessFixture(Fixture):
     to either filter the subprocess pipes or use the ``--logfile`` argument to write the output
     to a file in persistent storage.
 
+    Nanaimo's output processing may cause performance impact to subprocesses if output is sent 
+    from the subprocess faster than it can be processed by Nanaimo. In that case, the subprocess' 
+    output must be redirected to a file in order to prevent Nanaimo from attempting to process it.
```

Comment:
Hmm. I bet we could fix this by simply getting rid of the stdio pipes if no filters are provided. What do you think? Could you try that?

Reply:
That's a good idea. I'll give it a try but it might be a couple of days.

Reply:
Any updates on this?

Reply:
@thirtytwobits I experimented with changing line 799 of fixtures.py to the following (effectively disabling logging if there are no filters). That breaks the test "test_subprocess_fixture_logfile", because no logfile is created, because nothing is logged, because the test doesn't provide any filters.

I think this is a pretty big breaking change (no more default logging unless filters are passed)--maybe we should add an explicit option to not subscribe to the output?

```python
            if stderr_filter or stdout_filter:
                await self._wait_for_either_until_neither(
                    (proc.stdout if proc.stdout is not None else self._NoopStreamReader()),
                    (proc.stderr if proc.stderr is not None else self._NoopStreamReader()))

```